crates/formality-prove/src/decls.rs

```diff
@@ -117,8 +119,15 @@ pub struct NegImplDeclBoundData {
     pub where_clause: Wcs,
 }
 
-#[term(trait $id $binder)]
+#[term]
```

Comment:
Hmm, so if we wanted to do 

```rust
trait Foo { }
```

vs

```rust
unsafe trait foo { }
```

we have two ways to do it:

* Implement the parsing traits manually for `Safety` 
* Or extend the procedural macro to accommodate this case

I'm inclined to do the latter, as I think it will come up. Something like setting a "default" variant.

Reply:
> I'm inclined to do the latter, as I think it will come up. Something like setting a "default" variant.

I think that's reasonable; I honestly don't love the custom parsing strategy I took in this PR, and I'm happy to instead change the proc macro.